ansible/README.rst

```diff
@@ -0,0 +1,30 @@
+Ansible and Docker OVN Emulation Guide
+======================================
+
+Overview
+--------
+
+Deploy an OVN emulation environment using Docker container and ansible
```

Comment:
This tool can deploy an environment using Docker and ansible, how to run test cases on this environment, eg. create ports, please give a description about it.